xarray/plot/utils.py

```diff
@@ -490,7 +490,10 @@ def _get_units_from_attrs(da):
     else:
         units = _get_units_from_attrs(da)
 
-    return "\n".join(textwrap.wrap(name + extra + units, 30))
+    if name.startswith("$") and (name.count("$") % 2 == 0):
+        return "$\n$".join(textwrap.wrap(name + extra + units, 60))
+    else:
+        return "\n".join(textwrap.wrap(name + extra + units, 30))
```

Comment:
Shall we also set `break_long_words=False` here? https://docs.python.org/3/library/textwrap.html#textwrap.TextWrapper.break_long_words

Can you add a test to `test_plot.py`? It can be a standalone test that's outside the `TestPlot*` classes

Reply:
> Shall we also set break_long_words=False here? https://docs.python.org/3/library/textwrap.html#textwrap.TextWrapper.break_long_words

Probably a good idea! Just committed this but for some reason github is rejecting my push. Maybe servers are having issues?

Reply:
> Just create a long latex sequence, run it through label_from_attrs() and check if it's unbroken?

yes something like that.